### PR TITLE
Fallback to Aes-js if no aes-128-cfb8 in crypto

### PR DESCRIPTION
**Pull request by kukelekuuk**
for BoringSSL-based environments such as Electron

I did some limited testing on this, but I'm not entirely certain if it's a good implementation. I'll have to do more extensive testing at a later time.

### CODE REVIEW

File: src/transforms/encryption.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+  class Cipher extends Transform {
+    constructor (sharedSecret) {
+      super()
+      this.aes = new Cfb(sharedSecret, sharedSecret, 8)
+    }
+
+    _transform (chunk, encoding, callback) {
+      try {
+        callback(null, this.aes.encrypt(Buffer.concat([chunk, Buffer.alloc(chunk.length % 8)])))
```
Comment (rom1504) [inline src/transforms/encryption.js:17]: I don't see how this can work, this should be using a stream inside aes.js

Comment (rom1504) [reply src/transforms/encryption.js:17]: and why `chunk.length % 8` ?

**Comment by rom1504** [comment]
maybe you need https://github.com/ricmoo/aes-js/pull/97/files

**Comment by rom1504** [comment]
this didn't work at https://github.com/PrismarineJS/node-minecraft-protocol/pull/735/files 
anything new here @kukelekuuk ?

**Comment by kukelekuuk** [comment]
while I've made it work on a stream on my local version. But for some reason crypto's encrypt and aesjs' encrypt both give me different values. (also aesjs doesn't accept any values that aren't exactly a multiple of segment-size)

I'm not sure what to do with this for now. There doesn't appear to be a decent pure js implementation of cfb8 anywhere else. I'll close it and leave it alone for now.

**Pull request closed without merging**